sunkit_image/utils/tests/test_utils.py

```diff
@@ -108,10 +110,84 @@ def test_find_pixel_radii(smap):
     assert_quantity_allclose((np.max(pixel_radii)).value, known_maximum_pixel_radius)
 
     # Test that the new scale is used
-    pixel_radii = find_pixel_radii(smap, scale=2 * smap.rsun_obs)
+    pixel_radii = utils.find_pixel_radii(smap, scale=2 * smap.rsun_obs)
     assert_quantity_allclose(np.max(pixel_radii).value, known_maximum_pixel_radius / 2)
 
 
 def test_get_radial_intensity_summary():
     # TODO: Write some tests.
     pass
```

Comment:
What about this one?

Reply:
Oh yeah, I forgot about this. I'll add this.